sdk/forms/src/main/java/com/klaviyo/forms/KlaviyoWebView.kt

```diff
@@ -0,0 +1,50 @@
+package com.klaviyo.forms
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.webkit.WebView
+import androidx.webkit.WebViewCompat
+import androidx.webkit.WebViewFeature.WEB_MESSAGE_LISTENER
+import androidx.webkit.WebViewFeature.isFeatureSupported
+import com.klaviyo.analytics.DeviceProperties
+import com.klaviyo.core.BuildConfig
+import com.klaviyo.core.Registry
+import com.klaviyo.forms.KlaviyoWebViewDelegate.Companion.BRIDGE_NAME
+
+@SuppressLint("SetJavaScriptEnabled")
```

Comment:
KlaviyoWebViewDelegate seemed over-burdened. This was one obvious piece that could be extracted into its own class.